source/guide_codimd.rst

```diff
@@ -0,0 +1,303 @@
+.. highlight:: console
+.. author:: stunkymonkey <stunkymonkey.de>
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/codimd.png
+      :align: center
+
+######
+CodiMD
+######
+
+CodiMD_ lets you create real-time collaborative markdown notes on all platforms.
+Inspired by Hackpad, with more focus on speed and flexibility, and build from HackMD_ source code.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * MySQL_
+  * supervisord_
+  * domains_
+  * Node.js_ and its package manager npm_
+
+
+Prerequisites
+=============
+
+.. include:: includes/my-print-defaults.rst
+
+We're using Node.js_ in the stable version 8:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show node
+ Using 'Node.js' version: '8'
+ [isabell@stardust ~]$
+
+Your blog URL needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+You need a special database for CodiMD_:
+
+.. warning:: Replace ``<username>`` with your username!
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ mysql -e "CREATE DATABASE <username>_codimd CHARACTER SET utf8 COLLATE utf8_general_ci;"
+  [isabell@stardust ~]$
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ git clone https://github.com/hackmdio/codimd.git
```

Comment:
The project's README (https://github.com/hackmdio/codimd#installation) suggests downlading and extracting a release archive (https://github.com/hackmdio/codimd/releases) instead of cloning the current master